app.js

```diff
@@ -1,46 +1,46 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
+var express = require("express");
```

Comment:
You should consider using `let` and `const` instead of var, including for requires
There is also an ES6 import syntax, but im not entirely sure